Cargo.toml

```diff
@@ -19,9 +19,11 @@ rand = "0.7.3"
 log = "0.4.8"
 derivative = "2.1.1"
 uuid = "0.7.4"
+users = "0.10.0"
 
 [dev-dependencies]
 mockstream = "0.0.3"
 
 [features]
-testing = ["parsec-interface/testing"]
+testing = ["parsec-interface/testing", "no-security-check"]
+no-security-check = []
```

Comment:
I think the name is a bit extreme, maybe something like `no-fs-permission-check` - a bit lengthy but closer in meaning. Also, I guess it should be documented somewhere (both in Readme and in docs.rs?)

Reply:
Modified the name and added it to the README.